src/devices/Buzzer/README.md

```diff
@@ -0,0 +1,58 @@
+﻿
+# Buzzer
+
+  
+
+## Summary
+
+This device binding allow to play certain tone using piezo buzzer. It uses PWM with 50% duty cycle and various frequencies.
```

Comment:
"allow to play certain tone using piezo buzzer" -> "allow**s playing** certain tone**s** using piezo buzzer"

Reply:
Also fixed.